docs/themes/oso-tailwind/layouts/_default/list.html

```diff
@@ -1,46 +1,47 @@
-            {{ range $pages }}
-            {{ $page := . }}
-              <a href="{{ $page.RelPermalink }}"
-                class="m-2 border-opacity-20 hover:border-opacity-100 border-l-4 border-primary shadow-md rounded">
-                <div
-                  class="bg-primary-lightest text-primary-darkest w-full font-semibold text-lg pl-2 py-2 rounded-tr border-bottom border-gray-200">
-                  {{ $page.Title }}
-                </div>
-                <div class="px-2 pb-0.5 overflow-ellipsis overflow-hidden h-24">
-                  {{ with $page.Description }}
-                    <p>{{ . }}</p>
-                  {{ else }}
-                    <div>{{ $page.Plain | safeHTML }}</div>
-                  {{ end }}
-                </div>
-              </a>
+{{ if (or (ne .Language.Lang "any") (or .Params.any (and (not (isset .Params "any")) .Parent.Params.any))) }}
+<main id="content-wrapper" class="min-w-0 w-full flex-auto lg:static lg:max-h-full lg:overflow-visible">
+  <div class="w-full flex">
+    <div class="min-w-0 flex-auto px-4 sm:px-6 xl:px-8 pt-10 pb-24 lg:pb-16">
+      <article class="prose max-w-none min-w-0 flex-auto px-4 sm:px-6 xl:mr-64 xl:px-8 pt-10 pb-24 lg:pb-16">
+        {{ .Content }}
+      </article>
+      <!--/ Post Content-->
+      <div class="grid grid-cols-2 md:grid-cols-3 gap-2 justify-center py-4 leading-normal">
+        {{ $pages := .Pages }}
+        {{ if .Params.reverse }}{{ $pages = $pages.Reverse }}{{ end }}
+        {{ $pages = first 6 $pages }}
+        {{ range $pages }}
+        {{ $page := . }}
+        <a href="{{ $page.RelPermalink }}"
+          class="m-2 border-opacity-20 hover:border-opacity-100 border-l-4 border-primary shadow-md rounded">
+          <div
+            class="bg-primary-lightest text-primary-darkest w-full font-semibold text-lg pl-2 py-2 rounded-tr border-bottom border-gray-200">
+            {{ $page.Title }}
+          </div>
+          <div class="px-2 pb-0.5 overflow-ellipsis overflow-hidden h-24">
+            {{ with $page.Description }}
+            <p>{{ . }}</p>
+            {{ else }}
+            <div>{{ $page.Plain | safeHTML }}</div>
             {{ end }}
           </div>
-        </div>
-      </div>
-    </main>
-  {{ else }}
-    <div class="w-3/5 text-black leading-normal pt-6 mx-auto static">
-      <h2 class="text-xl font-semibold text-primary-dark pb-4">
-        Choose your language:
-      </h2>
-      <div class="grid mx-6 grid-cols-3 gap-8 content-evenly">
-        {{ partial "language-chooser.html" . }}
+        </a>
+        {{ end }}
       </div>
     </div>
-  {{ end }}
+  </div>
+</main>
+{{ else }}
+<div class="w-3/5 text-black leading-normal pt-6 mx-auto static">
+  <h2 class="text-xl font-semibold text-primary-dark pb-4">
+    Choose your language:
+  </h2>
+  <div class="grid mx-6 grid-cols-3 gap-8 content-evenly">
+    {{ partial "language-chooser.html" . }}
+  </div>
+</div>
+{{ end }}
+{{- partial "search.html" . -}}
```

Comment:
This is the only actual change in this file, right? Can you update your formatting settings to match the existing format? I'm not sure if we are using a consistent formatter for docs... so perhaps we should set that up to make diffs easier to review in the future.